### ISSUE TITLE
Grant Permissions dropdown isn't focusable

### ISSUE THREAD

**Issue by dananjohnson** [ManifoldScholar/manifold#2294]
![screencast 2019-07-25 14-55-29](https://user-images.githubusercontent.com/12454203/61911459-687c4400-aeec-11e9-9542-7d74bd11e88c.gif)

Consider refactoring as a native `<select>`.

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_
